### ISSUE TITLE
Fix interface/options for stochastic extended path in master

### ISSUE THREAD

**Issue by JohannesPfeifer** [DynareTeam/dynare#978]
Not all required options are set by default. For example, ```rs2.mod``` needs to manually set 
```
options_.ep.IntegrationAlgorithm='UT_2p+1';
``` 
as otherwise a crash in ```setup_integration_nodes.m``` happens. We should properly initialize all options and add a unit test that tests the default options.

**Comment by MichelJuillard** [comment]
The interface for extended path was never finished. I'm working on it currently

**Comment by JohannesPfeifer** [comment]
I was asking because of backward compatibility. The file at http://www.dynare.org/phpBB3/viewtopic.php?f=1&t=5220 does not work anymore (also, the question there needs to be answered)

**Comment by MichelJuillard** [comment]
Most of these options have never been documented as far as I know (except periods, solver_periods, order and hybrid) . So I don't see an issue of backward compatibility. The test cases will need to be changed to use the new interface.

**Comment by JohannesPfeifer** [comment]
Let me phrase it differently. All options should be initialized in ```global_initialization``` so that the baseline command runs without the user needing to specify additional options manually. This is currently not the case anymore. The referenced mod-file just uses the options you refer to
```
extended_path(periods=100,order=10);
```
but it still does not execute anymore.

**Comment by MichelJuillard** [comment]
I agree entirely. Redesigning the interface for extended path includes initializing the new options in global_initialization. I still have to see if it is possible to rationalize and standardize the option names used in various parts of the extended path code.
